images/compose/compose.folders.yml

```diff
@@ -42,16 +61,23 @@ services:
     command: ['pulp-api']
     depends_on:
       redis:
+        condition: service_healthy
       postgres:
         condition: service_healthy
     hostname: pulp-api
     user: pulp
     volumes:
       - "./assets/settings.py:/etc/pulp/settings.py:z"
-      - "./assets/certs:/etc/pulp/certs:z"
```

Comment:
Why are you removing the certs directory? Isn't this crucial for setting up cert auth and the CertGuard plugin?

Reply:
Would they auth and webserver certs not be in /etc/pki ?

Reply:
Also tls should be terminated by nginx and not pulp-api.